pdns/reczones.cc

```diff
@@ -237,15 +237,15 @@ static void makeNameToIPZone(const std::shared_ptr<SyncRes::domainmap_t>& newMap
 static void makeIPToNamesZone(const std::shared_ptr<SyncRes::domainmap_t>& newMap, const vector<string>& parts, Logr::log_t log)
 {
   string address = parts[0];
-  vector<string> ipparts;
-  stringtok(ipparts, address, ".");
+  vector<string> ipParts;
+  stringtok(ipParts, address, ".");
 
   SyncRes::AuthDomain ad;
   ad.d_rdForward = false;
 
   DNSRecord dr;
-  for (int n = ipparts.size() - 1; n >= 0; --n) {
-    dr.d_name.appendRawLabel(ipparts[n]);
+  for (auto part = ipParts.rbegin(); part != ipParts.rend(); ++part) {
```

Comment:
`const auto& part`

Reply:
This was removed later in favor of `ComboAddress`.